data/json/npcs/refugee_center/surface_refugees/NPC_Aleesha_Seward.json

```
@@ -143,7 +143,7 @@
     "dynamic_line": {
       "math": [ "time_since('cataclysm', 'unit':'days') >= 30" ],
       "yes": "We're just standing around here waiting, like a bunch of idiots.  We're supposedly waiting to go downstairs to the shelter, but it's been over a month.  I don't think it's happening.  I don't know what we're doing here.  I've read all the books, and there's zombies outside so we're stuck in here.  We can hear them at night.",
-      "no": "I don't know what's up.  I'm not sure what we've even doing here.  They say we're supposed to wait until we can be moved to the shelter downstairs, but we've been here days and there's no word on how long we'll be waiting.  It's all so stupid, and nobody can tell me anything.\"  She glances at the shuttered windows.  \"Jenny figures they're never going to let us go down there, so I guess we're going to sort it out up here, one way or another.  It's not fair, but 'fair' doesn't mean much in my life anymore."
+      "no": "I don't know what's up.  I'm not sure what we're even doing here.  They say we're supposed to wait until we can be moved to the shelter downstairs, but we've been here days and there's no word on how long we'll be waiting.  It's all so stupid, and nobody can tell me anything.\"  She glances at the shuttered windows.  \"Jenny figures they're never going to let us go down there, so I guess we're going to sort it out up here, one way or another.  It's not fair, but 'fair' doesn't mean much in my life anymore."
```

```suggestion
      "no": "I don't know what's up.  I'm not sure what we're even doing here.  They say we're supposed to wait until we can be moved to the shelter downstairs, but we've been here days and there's no word on how long we'll be waiting.  It's all so stupid, and nobody can tell me anything.\"  She glances at the shuttered windows.  \"Jenny figures they're never going to let us go down there, so I guess we're going to sort it out up here, one way or another.  It's not fair, but 'fair' doesn't mean much in my life anymore.",
```
Trying to fix broken test